content/en/platform/corda/5.3/reference/corda-cli/cpi.md

```diff
@@ -0,0 +1,81 @@
+---
+description: "List of CPI commands for the Corda CLI. Use these commands to upload or fetch CPIs."
+date: '2024-04-24'
+menu:
+  corda53:
+    identifier: corda53-cordacli-cpi
+    weight: 500
+    parent: corda53-cli-reference
+title: "cpi"
+---
+# cpi
+
+This section lists the Corda CLI `cpi` arguments. You can use these commands to upload CPIs to your project as well as list the CPIs that are already there.
+
+## upload
+
+The `upload` command uploads CPIs present in the specified `.cpi` file into your project.
```

Comment:
IMO "project" is a bit ambiguous, therefore proposing:
```suggestion
The `upload` command uploads CPIs present in the specified `.cpi` file into your C5 cluster, so that Virtual Nodes can be created using this CPI at a later point.
```